docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -310,14 +310,16 @@ Property name                         Description
 
 ``hive.s3.connect-timeout``           TCP connect timeout.                                        ``5 seconds``
 
+``hive.s3.connect-ttl``               TCP connect ttl, which affects connection reusage.          ``By default, connections do not expire.``
```

Comment:
> By default, connections do not expire.

This is actually no code. No need to highlight it.

Reply:
> > By default, connections do not expire.
> 
> This is actually no code. No need to highlight it with ``

Removed the `` highlight, thanks